.github/workflows/deploy.yml

```diff
@@ -8,9 +8,33 @@ on:
     branches: [dev, master]
 
 jobs:
+  check-diff:
```

Comment:
Is there any way to share `check-diff` between both files?

Reply:
We could try to make it is own action that is called by both files. I can give that a shot.